vtysh/vtysh.c

```diff
@@ -1326,6 +1326,13 @@ static struct cmd_node srv6_loc_node = {
 	.prompt = "%s(config-srv6-locator)# ",
 };
 
+static struct cmd_node srv6_prefix_node = {
+	.name = "srv6-locator-prefix",
+	.node = SRV6_PREFIX_NODE,
+	.parent_node = SRV6_LOC_NODE,
+	.prompt = "%s(config-srv6-locator-prefix)# ",
+};
+
```

Comment:
why defining a new srv6 prefix node ?
on a locator, we only have one prefix supported.
so I think it is not necessary to add a nesting level for sid commands.

many code change in this commit would be simplified. only the sid commands would be kept.

Reply:
as code moved to staticd, it is now necessary.